packages/financial-templates-lib/test/clients/OptimisticOracleEventClient.js

```diff
@@ -16,6 +16,11 @@ const Timer = getTruffleContract("Timer", web3);
 const Store = getTruffleContract("Store", web3);
 const MockOracle = getTruffleContract("MockOracleAncillary", web3);
 
+const objectsInArrayInclude = (superset, subset) => {
```

Comment:
same comments apply here

Reply:
See comments above, and let me know your thoughts.